x-pack/plugins/ml/public/jobs/jobs_list/components/delete_job_modal/delete_job_modal.js

```diff
@@ -79,26 +80,46 @@ export class DeleteJobModal extends Component {
       if (this.el && this.state.deleting === true) {
         // work around to disable the modal's buttons if the jobs are being deleted
         this.el.confirmButton.style.display = 'none';
-        this.el.cancelButton.textContent = 'Close';
+        this.el.cancelButton.textContent = (<FormattedMessage
+          id="xpack.ml.jobsList.deleteJobModal.closeButtonLabel"
+          defaultMessage="Close"
+        />);
       }
 
-      const title = `Delete ${(this.state.jobs.length > 1) ? `${this.state.jobs.length} jobs` : this.state.jobs[0].id}`;
+      const title = (
+        <FormattedMessage
+          id="xpack.ml.jobsList.deleteJobModal.deleteJobsTitle"
+          defaultMessage="Delete {jobsCount, plural, one {{jobId}} other {# jobs}}"
```

Comment:
Hi. It's a correct behavior for pseudo local, because pseudo locale translates all text in json file, values also are translated. You may run Kibana with custom locale and check that job ID isn't translated.
For example: I create a custom local fr.json and add to text $ symbol.  Job ID  isn't translated. 
![fr](https://user-images.githubusercontent.com/25555662/48468987-81c9de00-e7fe-11e8-90e0-c9df211f99dd.png)
![fr1](https://user-images.githubusercontent.com/25555662/48468993-842c3800-e7fe-11e8-951e-481871be9382.png)

Reply:
OK! Thanks for confirming job ID isn't translated.